plasma_framework/contracts/src/exits/payment/routers/PaymentStandardExitRouter.sol

```diff
@@ -67,11 +67,14 @@ contract PaymentStandardExitRouter is
         uint256 amount
     );
 
-    constructor(PaymentExitGameArgs.Args memory args)
+    function init(PaymentExitGameArgs.Args memory args)
         public
     {
+        require(!initDone, "Exit game was already initialized");
```

Comment:
I am a bit not sure if this is needed in Router but from past audit experience people prefer more but quite likely to be useless check so it might be good.

Please add unit test to the Routers test file too: `PaymentInFlightExitRouter.test` and `PaymentStandardExitRouter.test`